onnxruntime/hosting/request_handler.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "http_server.h"
+#include "json_handling.h"
+
+namespace onnxruntime {
+namespace hosting {
+
+namespace beast = boost::beast;
+namespace http = beast::http;
+
+void BadRequest(HttpContext& context, const std::string& error_message) {
```

Comment:
Better to keep the implementation in the `.cc` file.

Reply:
Moved.